mysqli/mysqli.php

```diff
@@ -947,11 +950,11 @@ class mysqli_result implements IteratorAggregate
 
     /**
      * Constructor (no docs available)
-     * @param object $mysql [optional]
+     * @param object $mysql
      * @param int $result_mode [optional]
      */
     public function __construct(
-        #[PhpStormStubsElementAvailable(from: '8.0')] mysqli $mysql = null,
+        #[PhpStormStubsElementAvailable(from: '8.0')] mysqli $mysql,
```

Comment:
Hm, `var_dump((new ReflectionMethod('mysqli_result::__construct'))->getNumberOfParameters());` returns 0.
All my `->getParameters()` tests are also returning nothing. Looks like this particular constructor doesn't have parameters or I'm missing something

Reply:
My bad, run in on < 8.0.